.goreleaser.yaml

```diff
@@ -0,0 +1,38 @@
+before:
+  hooks:
+    - go mod download
+builds:
+- id: monstache
+  binary: monstache
+  env:
+  - CGO_ENABLED=0
```

Comment:
I'm not entirely sure but this might mean that monstache go plugins would not work with these binaries.  That's probably ok though as that's a less common case.

Reply:
Ok I don't know how frequent is this usage. I can remove it if you want but since the image that runs gorelease is a Ubunutu, it will be compiled against glibc and not libmusl like you did before. Let me know what you want to do about this